core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -58,31 +58,240 @@ angular.module('oppia').factory('LiterallyCanvasHelperService', [
       }
     };
 
+    var validateSVGTag = function(svgString) {
```

Comment:
There's already functionality for this in the image-upload-helper service, so reuse that instead of duplicating it. Create a separate service that's focused on svg/image validation, if needed.

Reply:
Done.